### ISSUE TITLE
Authorize with Version v11.17.17 not working anymore

### ISSUE THREAD

**Issue by MWild67** [RSuter/NSwag#1425]
We migrated from v11.17.13 to v11.17.17 and now we can not Authorize anymore.

In v11.17.13 the ClientID and the Secret was already prefilled and we only had to select the API.
With v11.17.17 in the authorize dialog only the ClientID is shown and then the authorize fails.

**Comment by RSuter** [comment]
I have the feeling that this is related to this issue: https://github.com/RSuter/NSwag/issues/1419

?

**Comment by MWild67** [comment]
Yes, it seems that the Authorize type is always overwritten with implicit.

**Comment by MWild67** [comment]
Will you release this fix soon?
